packages/graphback-datasync/src/deltaHelper.ts

```diff
@@ -0,0 +1,46 @@
+import { ModelDefinition, getTableName } from '@graphback/core';
+import { Db, Collection, ObjectId } from "mongodb";
+import { DataSyncFieldNames } from './util';
+
+export function getDeltaTableName(tableName: string) {
+  return `${tableName}Delta`
+}
+
+/**
+ * M
+ */
+export class MongoDeltaHelper {
+  protected db: Db;
+  protected collectionName: string;
+  public constructor(model: ModelDefinition, db: Db) {
+    this.db = db;
+    this.collectionName = getDeltaTableName(getTableName(model.graphqlType));
+  }
+
+  public async insertDiff(updatedDocument: any) {
```

Comment:
How this will work if I:

Update just single field (userName) in the user object that has 10 fields?

Reply:
I am using the updated document to add the diff:
https://github.com/aerogear/graphback/blob/601fe9f40cadf9ee5617073f491d05052d6ed9e5/packages/graphback-datasync/src/providers/DataSyncConflictProvider.ts#L74-L76
Should I change that?

Reply:
No. just  my bad. All good here